tao_compiler/mlir/disc/transforms/element_type_converter.cc

```diff
@@ -157,6 +157,38 @@ struct ConvertReduceOpWithSmallWidthIntType
     return success();
   }
 };
+struct ConvertDynamicConvOp : public OpRewritePattern<mhlo::DynamicConvOp> {
+  using OpRewritePattern<mhlo::DynamicConvOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(mhlo::DynamicConvOp op,
```

Comment:
What about mhlo::ConvOp?

Reply:
May use template to deal with both ConvOp and DynamicConvOp:
`
template <typename OpTy>
struct ConvertConvOp : public OpRewritePattern<OpTy> {
...;
dyn_cast ...;
}
`

Reply:
Seems like DynamicConvOp has one more operand than ConvOp.

Reply:
Done.